core-ui/src/components/Predefined/Details/Certificate/CertificateRefs.js

```diff
@@ -1,12 +1,30 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
-import { LayoutPanel, FormItem, FormLabel } from 'fundamental-react';
+import { LayoutPanel, FormItem, FormLabel, Token } from 'fundamental-react';
+import { EMPTY_TEXT_PLACEHOLDER, Labels } from 'react-shared';
 
 import { IssuerLink } from './IssuerLink';
 import { SecretLink } from './SecretLink';
 
 import './CertificateRefs.scss';
 
+const Tokens = ({ tokens }) => (
+  <div>
+    {tokens.length
+      ? tokens.map(scope => (
+          <Token
+            key={scope}
+            buttonLabel=""
+            className="y-fd-token y-fd-token--no-button y-fd-token--gap fd-margin-end--tiny"
+            readOnly={true}
+          >
+            {scope}
+          </Token>
+        ))
+      : EMPTY_TEXT_PLACEHOLDER}
+  </div>
+);
+
```

Comment:
I've noticed it's the same code as in `core-ui/src/components/Predefined/Details/OAuth2Clients/OAuthClientSpecPanel.js`. Why don't we extract them to separate file?